deploy/crds/devconsole_v1alpha1_gitsource_crd.yaml

```diff
@@ -86,6 +86,10 @@ spec:
                   description: Error has the error message if the attempt to
                     reach a GitSource failed
                   type: string
+                reason:
+                  description: Reason is a short predefined string representing
```

Comment:
it would be great if we could limit it by some parameter, but the only usable validation parameter I'm aware of is the `pattern`. There is no other right?
I'll add it to the description and let me know if you want me to add the pattern there as well.
What would the best for UI @christianvogt?

Reply:
> it would be great if we could limit it by some parameter, but the only usable validation parameter I'm aware of is the pattern. There is no other right?

Right!